### PR TITLE
Change syntax for set membership predicate ('in' -> 'contains')

### PR DESCRIPTION
**Pull request by AltGr**
Also includes a small CLI patch

### CODE REVIEW

**Comment by AltGr** [comment]
I am surprised that only 1 test and no examples are affected by the change :raised_eyebrow:

**Comment by AltGr** [comment]
Update: replaced "comprend" by "contient" for the french version, and got a :heavy_check_mark: from the legal team

File: compiler/surface/lexer_pl.cppo.ml [hunk 1]
```diff
@@ -29,6 +29,7 @@
 #define MS_INCREASING "rosnacy"
 #define MS_OF "z"
 #define MS_COLLECTION "kolekcja"
+#define MS_CONTAINS "contains"
```
Comment (denismerigoux) [inline compiler/surface/lexer_pl.cppo.ml:32]: The translation of `contains` in Polish seems to be "[zawiera](https://context.reverso.net/translation/polish-english/zawiera)"

**Review by denismerigoux — APPROVED** [review]
All good, thanks ! Except the Polish translation, we should keep this frontend updated while we can.

File: compiler/surface/lexer_pl.cppo.ml (hunk 1 above)
Comment (AltGr) [reply compiler/surface/lexer_pl.cppo.ml:32]:
```suggestion
#define MS_CONTAINS "zawiera"
```

**Pull request merged**